### PR TITLE
Chore: Migrate edxapp docker build

### PR DESCRIPTION
**Pull request by michaelroudnitski**
This will probably be the largest one.

I excluded the **removal** of [ansible_overrides.yml](https://github.com/ibm-skills-network/configuration/blob/open-release/juniper.master/docker/build/edxapp/ansible_overrides.yml). We can remove it if necessary but I wasn't sure if we needed to. The Juniper ansible_overrides contains some vars that weren't present in our ironwood fork.

### CODE REVIEW

File: docker/build/edxapp/docker-run.sh [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  # extract the first 3 characters to find the appropriate service (lms or cms)
+  local service=${1:0:3}
+  [[ ${1:3} == "-workers" ]] && local workers=true
+
+  # run edxapp playbook to setup config files
+  cd /edx/app/edx_ansible/edx_ansible/docker/plays
+
+  if [[ -z "$RELEASE_NAME" ]]; then
+    /edx/app/edx_ansible/venvs/edx_ansible/bin/ansible-playbook \
+        edxapp-run.yml \
```
Comment (michaelroudnitski) [inline docker/build/edxapp/docker-run.sh:70]: (also line 79)

may need to be changed to `edxapp.yml` for the reason described here https://github.com/ibm-skills-network/configuration/pull/7#issue-520890012

File: docker/build/edxapp/vars/build.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# EDXAPP_MONGO_HOSTS:
+#   - "mongo"
+EDXAPP_MEMCACHE: ["memcached"]
+EDXAPP_PYTHON_SANDBOX: false
+
+# Git configuration
+EDXAPP_GIT_IDENTITY: "{{ BDU_PORTAL_DEPLOY_KEY }}"
+EDXAPP_USE_GIT_IDENTITY: true
+edx_platform_repo: git@github.ibm.com:skills-network-portals/edx-platform.git
+edx_platform_version: open-release/ironwood.master
```
Comment (danielrudn) [inline docker/build/edxapp/vars/build.yml:28]: The repo should point to github.com/edx/edx-platform, which should be the default.

The version should point to `open-release/juniper.master`

**Review by danielrudn — CHANGES_REQUESTED** [review]

File: docker/build/edxapp/vars/build.yml (hunk 2 above)
Comment (michaelroudnitski) [reply docker/build/edxapp/vars/build.yml:28]: Should we also change lines 25, 36 & 42?

Comment (danielrudn) [reply docker/build/edxapp/vars/build.yml]: No, those should still point to GHE

**Pull request merged**